README.md

```diff
@@ -62,7 +62,9 @@ Semantic HTML is the use of HTML markup to reinforce the semantics, or meaning,
 * How would you optimize a website's assets/resources?
 * How many resources will a browser download from a given domain at a time?
   * What are the exceptions?
-* Name 3 ways to decrease page load (perceived or actual load time).
+* Name 3 ways to decrease page load (perceived or actual load time).  
+1. Optimalize images - decrease their amount, use miniatures, svg, css-sprites, preloaders 2.Cache content 3. Remove FOUC (flash of unstyled content)  
```

Comment:
Dodałem to bo w pytaniu było również perceived load time (szczerze to nie jestem przekonany czy to rzeczywiście zmniejsza perceived load time, ale w odpowiedziach od Ciebie ktoś to zawarł). Było też zmniejszenie ilości requestów http, podmienić?